src/App/Pages/Lock/LockFingerprintPage.cs

```diff
@@ -32,9 +33,23 @@ public LockFingerprintPage(bool checkFingerprintImmediately)
 
         public void Init()
         {
+            var image_helper = Helpers.OnPlatform(
+                        iOS: _deviceInfoService.HasFaceIdSupport ? "smile.png" : "fingerprint.png",
+                        Android: _deviceInfoService.HasFaceIdSupport ? "smile.png" : "fingerprint.png",
```

Comment:
Android does not have faceid support, so just leave this as `fingerprint.png` only.